noir_stdlib/src/sha256.nr

```diff
@@ -100,30 +100,26 @@ fn msg_u8_to_u32(msg: [u8; 64]) -> [u32; 16]
 
 // SHA-256 hash function
 #[alternative(sha256)]
-fn digest(msg: [u8]) -> [u8; 32]
-{
+fn digest(msg: [u8]) -> [u8; 32] {
```

Comment:
Why wasn't `N` added to this like in the sha256 examples above?

Reply:
It was forgotten about since the change was minor. I'll update it